spec/support/fixtures/bookshelf/config/environment.rb

```diff
@@ -65,6 +66,7 @@ module Metrics
       manifest         "assets.json"
       prefix           "/assets/metrics"
       compile          true
+      nested           true
```

Comment:
```suggestion
      nested           false
```

Reply:
I addressed both of these in a single commit (ffbc598)